_posts/2022-11-20-jump_dev_2023.md

```diff
@@ -26,6 +26,18 @@ will open in early 2023.
 
 All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2023/coc/).
 
+## Travel support
+
+To encourage people to attend JuMP-dev, we have funds to provide travel support
+to a limited number of applicants.
+
+We particularly encourage people who have not been to a JuMP-dev before to
+apply. **You do not need to give a talk to apply for travel support.**
```

Comment:
Language needs to be updated to match the form

Reply:
Removed the sentence.